### DISCUSSION TITLE
ThunderClient behaving slow in my machine.

### DISCUSSION THREAD

**Discussion by mubeentajss** [thunderclient/thunder-client-support#1345] [Q&A]
Hello All,

I installed thunder client extension to Vscode, but I encounter performance issue it takes few seconds to click on each tab for every API request field and its not loading the payload data in few seconds, it takes time to see the output response as well, kindly help on this issue i tried to re-install thunder client to new version, I disabled all the extension from Vscode, I checked the memory usage as well it all seems to be normal but I still face this issue when I click on each tab of header ,auth, body from thunderClient it takes few seconds to load and respond this makes my work delayed, Please suggest on this & thanks in advance.

**Comment by rangav** [comment]
Hi @mubeentajss 

can you share extension version 

and please share screenshots or record a video

**Reply by mubeentajss** [reply]
Its not possible to share recording from office laptop, when I click on header tab,Auth tab,tests tab it wont load quickly it will take few seconds and then I can able to see the content till that it will be blank

**Reply by rangav** [reply]
Some times this may happen. Can you try reload vscode and try again.

**Reply by mubeentajss** [reply]
I did reloading multiple time but still tab navigation takes few seconds to load content

**Reply by rangav** [reply]
Try installing previous version of extension

**Reply by mubeentajss** [reply]
let me do that

**Reply by mubeentajss** [reply]
Now am facing once I click on API it will take some time to convert from mouse arrow symbol to hand moment symbol but once loaded in few seconds tab clicking content is loading

**Reply by rangav** [reply]
Thats strange, please record a video and share to better under your problem.

**Reply by mubeentajss** [reply]
I still see performance issue in it is taking few seconds 30 to 40 seconds to load content

**Reply by mubeentajss** [reply]
is there a way to check the thunderclient logs for this loading issue, I need to get a approvals to share a video

**Reply by rangav** [reply]
https://github.com/rangav/thunder-client-support#logs

**Reply by mubeentajss** [reply]
I can see in logs : Settings Data:
saveToWorkspace: undefined
workspaceRelativePath: 
multiRootWorkspaceName: 
customLocation: undefined

**Reply by rangav** [reply]
Keep trying installing all previous versions and see which one works

**Reply by mubeentajss** [reply]
okay cool let me do this way as well appreciate your help and support I'll update you shortly

**Reply by mubeentajss** [reply]
and also one another issue I was facing was, one of the API call is executing infinite time and thunderClient unable to send response output...while in postman same am executing it is showing  respond after few minutes...is there a way to fix this issue only for specific API call am facing this issue from long time

**Reply by mubeentajss** [reply]
the 5 months ago version worked, thanks for your support on this issue I'll monitor this for few days if its consistent then I think am good to stick with this version.

**Reply by mubeentajss** [reply]
could you help on API timeout infinite time loading I mean its spinning for longer time.

**Reply by rangav** [reply]
It is illegal to use older version according to our terms, so please update to latest version in the coming days

**Reply by mubeentajss** [reply]
Latest version is not supporting in my machine at the moment.

**Reply by rangav** [reply]
use any other tool that works

**Comment by mubeentajss** [comment]
here is the extension details : Thunder Client v2.12.1

**Comment by mubeentajss** [comment]
Now am facing once I click on API it will take some time to convert from mouse arrow symbol to hand moment symbol but once loaded in few seconds tab clicking content is loading
